python/cuml/cuml/model_selection/__init__.py

```diff
@@ -13,19 +13,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
-from sklearn.model_selection import GridSearchCV
-
-from cuml.model_selection._split import StratifiedKFold, train_test_split
-
-GridSearchCV.__doc__ = (
-    """
-This code is developed and maintained by scikit-learn and imported
+"""This code is developed and maintained by scikit-learn and imported
```

Comment:
We no longer patch the docstrings of `GridSearchCV` (or `Pipeline` below). Before this was _mutating_ the docstring of the sklearn equivalent (since we use the same exact class), which would result in the docstring of `sklearn.model_selection.GridSearchCV` being different after importing `cuml`. This seems wrong.

_IF_ we still want a disclaimer on the class itself, we should subclass just to add the docstring. I think tossing a small module level docstring like this though should be sufficient. Or we should maybe consider deprecating and removing the `model_selection`/`pipeline` modules entirely since upstream has that handled.

Reply:
For sure, was just noting this change for discussion.